pygmt/src/select.py

```diff
@@ -128,6 +131,38 @@ def select(data=None, outfile=None, **kwargs):
         and **+i** reverses the tests to pass record with *z* value NOT in the
         given range. Finally, if **+c** is not used then it is automatically
         incremented for each new ``z_subregion`` argument, starting with 2.
+    dist2pt : str
```

Comment:
Could you move the docstring for dist2pt/dist2line/polygon up a bit? Maybe under `area_thresh`?

Reply:
This is done in commit https://github.com/GenericMappingTools/pygmt/pull/2466/commits/ba6c65314c797dab431191b0686919ed7b0a0edd.